dll/win32/CMakeLists.txt

```diff
@@ -42,7 +42,9 @@ add_subdirectory(faultrep)
 add_subdirectory(fltlib)
 add_subdirectory(fmifs)
 add_subdirectory(fontsub)
-add_subdirectory(framedyn)
+if(NOT USE_CLANG_CL)
+    add_subdirectory(framedyn)
```

Comment:
why?

Reply:
Tons of unresolved references and I'm not sure why clang is not picking them up. We may need some trickery like we're currently doing for gcc here.

Reply:
In that case it makes sense to create a jira issue and reference that here in a comment.

Reply:
Sounds good.

Reply:
Framedyn exports C++ symbols so it has lots of assembly redirectors/wrappers and a complicated .spec file. It makes sense that this will need work.